Wireframe/css/style.css

```diff
@@ -0,0 +1,141 @@
+:root {
+  --font-family: sans-serif;
+  --orange-dark: #ef6419;
+  --aqua-color: aqua;
+  --grey-color: grey;
+}
+
+html,
+body {
+  box-sizing: border-box;
+  background-color: #ffeead;
+  height: 100%;
+  padding: 10px;
+  margin: 0px;
+}
+
+.container {
+  height: 100%;
+  display: grid;
+  grid-gap: 6px;
+  /* grid-template-columns: [main-start] 1fr [content-start] 5fr [content-end main-end];
+  grid-template-rows: [main-start] 40px [content-start] auto [content-end] 40px [main-end]; */
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: 40px 510px 50px 10px;
+  grid-template-areas:
```

Comment:
Everything you write as code should make things clearer, not harder to understand. Good code is clear and simple.

Don't use letters, use the actual words. There's no benefit to using a letter - it obscures your meaning for other developers, and in production all your code is automatically compressed and minified, so you're not saving any bytes this way.